### PR TITLE
Add support for "--health" as an alias for the Meta::health method

### PR DESCRIPTION
**Pull request by prashantv**
@abhinav @akshayjshah

### CODE REVIEW

File: meta_test.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetMetaFile(t *testing.T) {
+	f, err := getMetaFile()
+	require.NoError(t, err, "getMetaFile failed")
```
Comment (abhinav) [inline meta_test.go]: You probably want to delete the file afterwards

File: options.go [hunk 2]
```diff
@@ -31,10 +31,11 @@ type Options struct {
 
 // RequestOptions are request related options
 type RequestOptions struct {
-	ThriftFile  string `short:"t" long:"thrift" description:"Path of the .thrift file" required:"true"`
+	ThriftFile  string `short:"t" long:"thrift" description:"Path of the .thrift file"`
 	MethodName  string `short:"m" long:"method" description:"The full Thrift method name (Svc::Method) to invoke"`
 	RequestJSON string `short:"r" long:"request" description:"The request body, in JSON format"`
 	RequestFile string `short:"f" long:"file" description:"Path of a file containing the request body in JSON"`
+	Health      bool   `short:"h" long:"health" description:"Hit the health endpoint, Meta::health"`
```
Comment (abhinav) [inline options.go]: Will `-h` not conflict with the help flag? In fact most people will expect `-h` to be help.

**Comment by abhinav** [comment]
LGTM besides minor things

File: meta_test.go (hunk 1 above)
Comment (prashantv) [inline meta_test.go]: :+1:

File: options.go (hunk 2 above)
Comment (prashantv) [inline options.go]: Good point, I'll remove `-h` and only leave the long option for now.

**Pull request merged**